.github/workflows/test.yml

```
@@ -0,0 +1,64 @@
+name: test
+
+on:
+  push:
+    branches:
+      # - master
+  pull_request:
+
+jobs:
+  build:
```

```suggestion
name: CI

on:
  push:
    branches:
      - main
      - master
  pull_request:

jobs:
  test:
```

Updates the on push so the tests don't run twice, and changed to standardize the ci naming convention across our gems

---

Why would this run twice without it?